arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/CallSection.java

```diff
@@ -630,11 +636,11 @@ private void completeSmcOrPrcSuccessWillRevert(Hub hub) {
       scenarioFragment.setScenario(CALL_PRC_SUCCESS_WILL_REVERT);
     }
 
-    callerSecond = callerFirstNew.deepCopy().setDeploymentInfo(hub);
-    callerSecondNew = callerFirst.deepCopy().setDeploymentInfo(hub);
+    callerSecond = callerFirstNew.deepCopy().setDeploymentNumber(hub);
+    callerSecondNew = callerFirst.deepCopy().setDeploymentNumber(hub);
 
-    calleeSecond = calleeFirstNew.deepCopy().setDeploymentInfo(hub);
-    calleeSecondNew = calleeFirst.deepCopy().setDeploymentInfo(hub);
+    calleeSecond = calleeFirstNew.deepCopy().setDeploymentNumber(hub);
+    calleeSecondNew = calleeFirst.deepCopy().setDeploymentNumber(hub);
```

Comment:
The deployment number may have evolved, but it is the deployment status we are walking back.